### PR TITLE
CHAOS-5230 GameDay V2

### PR DESCRIPTION
**Pull request by SmritiSatya**
[GameDay V2](https://663378c25ac478bc2c6213f9--harness-developer.netlify.app/docs/chaos-engineering/features/gameday/gameday-v2)

### CODE REVIEW

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+title: Run a GameDay (V2)
+sidebar_position: 3
+description: Guide to creating and executing a GameDay
+---
+
+This section describes how to create and execute [GameDay](/docs/chaos-engineering/features/gameday/introduction-to-gameday.md) V2.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:7]:
```suggestion
This topic describes how to create and execute [GameDay](/docs/chaos-engineering/features/gameday/introduction-to-gameday.md) V2.
```

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+### Add stakeholder and role
+
+2. Add stakeholder names, their roles, and whether they are allowed to approve a GameDay run. Only a stakeholder can approve or reject a GameDay run. Select **Next: Select Prerequisites**.
+
+ ![](./static/v2/add-stakeholder-3.png)
+
+### Create prerequisites
+
+You can create prerequisites for your GameDay that would:
```
Comment (brian-f-harness) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:33]:
```suggestion
You can create prerequisites for your GameDay that:
```

File: docs/chaos-engineering/features/gameday/run-gameday.md [hunk 3]
```diff
@@ -1,94 +1,105 @@
...
+title: Run a GameDay (V1)
 sidebar_position: 2
-description: Guide to create and execute a GameDay
+description: Guide to creating and executing a GameDay
 redirect_from:
    - /docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday
 ---
 
-Learn how to create and execute a [GameDay](/docs/chaos-engineering/features/gameday/introduction-to-gameday.md).
+This section describes how to create and execute a [GameDay](/docs/chaos-engineering/features/gameday/introduction-to-gameday.md).
```
Comment (brian-f-harness) [inline docs/chaos-engineering/features/gameday/run-gameday.md:9]:
```suggestion
This topic describes how to create and execute a [GameDay](/docs/chaos-engineering/features/gameday/introduction-to-gameday.md).
```

File: docs/chaos-engineering/features/gameday/run-gameday.md [hunk 4]
```diff
@@ -1,94 +1,105 @@
...
+Address the following questions before proceeding to run a GameDay:
 
 1. Which services should I test?
 2. What is the goal of the GameDay?
 3. What should I verify or determine by the end of the GameDay?
 
 Once these questions have a viable answer, you can proceed to create a GameDay.
 
-## Create a GameDay and specify the details
+If you are new to HCE, head to [GameDay V2](/docs/chaos-engineering/features/gameday/gameday-v2.md) and follow the steps to create and run a GameDay
```
Comment (brian-f-harness) [inline docs/chaos-engineering/features/gameday/run-gameday.md:26]:
```suggestion
If you are new to HCE, head to [GameDay V2](/docs/chaos-engineering/features/gameday/gameday-v2.md) and follow the steps to create and run a GameDay.
```

File: docs/chaos-engineering/features/gameday/run-gameday.md [hunk 5]
```diff
@@ -1,94 +1,105 @@
...
+9. Now that you have the experiments in the GameDay, you are all set to schedule or run them.
 
    ![](./static/run-gameday/10-gameday-created.png)
 
-2. Click **Start GameDay**. This creates experiments within your GameDay.
+10. Click **Start GameDay**. This creates experiments within your GameDay.
+
+      ![](./static/run-gameday/11-start-gameday.png)
+
+11. Clicking GameDay leads you to the page that contains the experiments associated with the GameDay.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/features/gameday/run-gameday.md:80]:
```suggestion
11. Clicking **GameDay** opens the page that contains the experiments associated with the GameDay.
```

**Review by brian-f-harness — APPROVED** [review]

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+### Create a GameDay
+
+1. Select **+New GameDay**.
+
+ ![](./static/v2/click-gameday-1.png)
+
+### Add stakeholder and role
+
+2. Add stakeholder names, their roles, and whether they are allowed to approve a GameDay run. Only a stakeholder can approve or reject a GameDay run. Select **Next: Select Prerequisites**.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:27]: allowed -> required (as anyone can approve/reject the run on the behalf of selected stakeholder).
Only a stakeholder can approve or reject a GameDay run -> As mentioned above anyone can approve/reject on others' behalf.

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+ * Validate environment health.
+ * Check the availability of relevant chaos tools.
+ * Check if the chaos tools are configured with the right chaos artefact sources.
+ * Check the availability of the team members.
+
+3. Select **+ Add prerequisite**.
+
+ ![](./static/v2/create-prereq-4.png)
+
+4. Enter a name and a description (optional). Select **Save**.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:43]: Instead of name, we can rephrase it to something like - title for the prerequisite.

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+:::tip
+* Select the experiment to view the details of the selected experiment.
+
+ ![](./static/v2/select-exp-11.png)
+:::
+
+### Check prerequisites
+
+8. Once the stakeholder approves the GameDay run, you can proceed.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:69]: stakeholder approves -> stakeholders approve (as there can be multiple stakeholders)

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+6. Select an environment, an infrastructure type, an infrastructure, and a ChaosHub. Select the chaos experiment and select **Add Experiment to GameDay**.
+
+ ![](./static/v2/select-experiments-9.png)
+
+7. Select **Save** (or **Discard** if you don't want to proceed). You can edit or delete experiments in this step.
+
+ ![](./static/v2/save-gameday-10.png)
+
+:::tip
+* Select the experiment to view the details of the selected experiment.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:62]: This is not available currently, but should be available in the upcoming releases

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+- A stakeholder can reject the run and state reasons in the comments.
+
+ ![](./static/v2/add-or-edit-14.png)
+:::
+
+9. Select **Next: Check prerequisites** to validate whether the prerequisites you mentioned earlier have been fulfilled.
+
+ ![](./static/v2/check-prereq-15.png)
+
+10. HCE checks if the prerequisites are fulfilled. If one of them is not fulfilled, you will not be able to execute the experiments.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:83]: HCE doesn't check, users have to mark the prerequisites as done to move forward

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+12. Once all the experiments complete execution, select **Done Running Experiments**.
+
+ ![](./static/v2/done-exp-21.png)
+
+:::info note
+- Select **Terminate GameDay Run** to stop GameDay execution. Select **Terminate this run** to confirm.
+
+ ![](./static/v2/terminal-run-29.png)
+
+- Select **Add/View Notes** to note your findings about the GameDay.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:107]: This notes is on experiment level, users can add notes related to those particular experiments. gameday run level notes can be added in the post mortem step

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+
+:::info note
+- Select **Done Running Experiments** to complete the run. If your chaos experiments have faults that have not run, they will be marked **Skipped**.
+
+ ![](./static/v2/done-running-28.png)
+:::
+
+### Download report
+
+14. Select **Download Run Report** to download the report.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:124]: Not available yet

File: docs/chaos-engineering/features/gameday/gameday-v2.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+
+ ![](./static/v2/download-27.png)
+
+### View experiment execution
+
+- Select the experiment in the GameDay to view manifest details. Select **View Execution** to see more.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/features/gameday/gameday-v2.md:130]: not available yet

**Review by Saranya-jena — CHANGES_REQUESTED** [review]
